internal/cmd/commands/rolescmd/funcs.go

```diff
@@ -207,20 +205,6 @@ func extraFlagsFuncImpl(c *Command, _ *base.FlagSets, f *base.FlagSet) {
 
 func extraFlagsHandlingFuncImpl(c *Command, _ *base.FlagSets, opts *[]roles.Option) bool {
 	switch c.Func {
-	case "create", "update":
-		switch len(c.flagGrantScopeIds) {
-		case 0:
-		case 1:
-			if c.flagGrantScopeIds[0] == "null" {
-				*opts = append(*opts, roles.DefaultGrantScopeId())
-			} else {
-				*opts = append(*opts, roles.WithGrantScopeId(c.flagGrantScopeIds[0]))
-			}
-		default:
-			c.UI.Error("-grant-scope-id cannot be specified multiple times when using the deprecated create/update grant_scope_id mechanism")
-			return false
-		}
```

Comment:
Sanity check: we're ok to remove this now, right? This flag isn't supported for `create` and `update` anymore.